README.md

```diff
@@ -206,7 +193,32 @@ wiki page.
    Additionally, the cause of the "PCM not found" error might be an incorrect ALSA PCM name. Run
    `bluealsa-aplay --list-pcms` for the list of currently available ALSA PCM names - it might give
    you a hint what is wrong with your `.asoundrc` entry. Also, take a look at the
-   [bluealsa-plugins man page](./doc/bluealsa-plugins.7.rst).
+   [bluealsa-plugins manual page](./doc/bluealsa-plugins.7.rst).
+
+## Contributing
+
+This project welcomes contributions of code, documentation and testing. For code and manual page contributions, please use Github Pull Requests. There is no strict policy for PRs, each contribution will be evaluated individually. If you wish to help by testing PRs or by making review comments please do so by adding comments to the PR.
+
+When preparing a pull request, if possible please configure with `--enable-test`, and to catch as many coding errors as possible please compile with:
+```
+make CFLAGS="-Wall -Wextra -Werror"
+```
+and then run the unit test suite:
+```
+make check
+```
+
+The project wiki is "public" and contributions there are also welcome.
+
+## Bug reports, feature requests, and requests for help
+
+Please search previous issues (both open and closed), to see if your question has already been answered or problem resolved, before raising a new issue. If reporting a problem, please clearly state the version of BlueALSA that you are using, and give sufficient information for readers to be able to reproduce the issue.
+
+Please also look at the [wiki](https://github.com/Arkq/bluez-alsa/wiki) if you require help as there is a great deal of useful information there. Unfortunately the wiki is not indexed by web search engines, so searching on-line for your issue will not discover the information in there.
```

Comment:
Double "there" in the sentence sounds strange to my non-native English ears :D Maybe the last one can be dropped?

```suggestion
Please also look at the [wiki](https://github.com/Arkq/bluez-alsa/wiki) if you require help as there is a great deal of useful information. Unfortunately the wiki is not indexed by web search engines, so searching on-line for your issue will not discover the information in there.
```

Reply:
bluez-alsa github has just reached 700 stars, which appears to be the threshold for allowing search engine robots in. So the issues list is now indexed by Google etc :champagne: